app/uploaders/image_uploader.rb

```diff
@@ -0,0 +1,24 @@
+# encoding: utf-8
+
+class ImageUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  include CarrierWave::RMagick
+
+  # Choose what kind of storage to use for this uploader:
+ if Rails.env.test?
+   storage :file
```

Comment:
The test for which storage solution to use should be in `config/initializers/fog.rb`, and the storage lines should be completely removed from here.